exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -202,7 +207,85 @@ def export(
         **kwargs,
     ) -> MetricExportResult:
         # TODO(#2663): OTLPExporterMixin should pass timeout to gRPC
-        return self._export(metrics_data)
+        if not self.max_export_batch_size:
```

Comment:
```suggestion
        if self._max_export_batch_size is None:
```

Reply:
Actually, `math.inf` should be a better default value that should make this check unnecessary.

Reply:
I kept the default value to None to avoid doing any computation when this feature is not used.